ad7768-1fmcz/ad77681.c

```diff
@@ -134,12 +155,16 @@ int32_t ad77681_spi_reg_write(struct ad77681_dev *dev,
 			      uint8_t reg_addr,
 			      uint8_t reg_data)
 {
-	uint8_t buf[2];
+	uint8_t buf[3];
```

Comment:
This change has no business, with the changes in `ad77681_compute_crc8()`function.
This commit, should be split in multiple commits, by functionalities that it touches.